app/javascript/packs/public.js

```diff
@@ -3,6 +3,7 @@ import { getLocale } from 'mastodon/locales';
 import { length } from 'stringz';
 import IntlRelativeFormat from 'intl-relativeformat';
 import { delegate } from 'rails-ujs';
+import loadPolyfills from '../mastodon/load_polyfills';
```

Comment:
I agree. It may be good to rewrite other parts as well. However, irregularities are not good.